Implement currentStackPointer for Linux/LoongArch64

xiangzhai opened the pull request:
#### f27ad76123d487f483435fa5629744d45042daa4
<pre>
Implement currentStackPointer for Linux/LoongArch64
<a href="https://bugs.webkit.org/show_bug.cgi?id=268131">https://bugs.webkit.org/show_bug.cgi?id=268131</a>

Reviewed by Mark Lam and Yusuke Suzuki.

Implement currentStackPointer for Linux/LoongArch64 to fix the build error.

* Source/WTF/wtf/StackPointer.cpp:

Canonical link: <a href="https://commits.webkit.org/276739@main">https://commits.webkit.org/276739@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/6c104bd74d8b410bc4c550c78cb7e035e1a12245

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/37769 "1 style error") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/16664 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/40010 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/40309 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/33600 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/19312 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/13887 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/31954 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/38336 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/14019 "Found 1 new test failure: media/video-ended-does-not-hold-sleep-assertion.html (failure)") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/33061 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/12253 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/12187 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/33758 "Passed tests") | [⏳ 🛠 wpe-skia ](https://ews-build.webkit.org/#/builders/WPE-Skia-Build-EWS "Waiting in queue, processing has not started yet") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/31491 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/34168 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/34188 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/41568 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/37473 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/12782 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/10345 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/38074 "Found 1 new test failure: imported/w3c/web-platform-tests/preload/prefetch-document.html (failure)") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/14195 "Built successfully") | | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/36249 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/13163 "Built successfully") | | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/44382 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/6333 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/13506 "Built successfully") | | [❌ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/9065 "Found 2 new JSC stress test failures: wasm.yaml/wasm/gc/bug266056.js.wasm-eager, wasm.yaml/wasm/gc/const-exprs-flag-off.js.wasm-eager (failure)") | 
<!--EWS-Status-Bubble-End-->

Constellation reviewed (changes requested):
Thanks. Please follow to https://webkit.org/contributing-code/ to use the defined commit message format etc. (And use `git-webkit` command to update PR). WebKit commit requires specific commit message format & PR format.

MenloDorian reviewed (approved):
r=me

Constellation reviewed (approved):
r=me

xiangzhai committed:
Implement currentStackPointer for Linux/LoongArch64
https://bugs.webkit.org/show_bug.cgi?id=268131

Reviewed by Mark Lam and Yusuke Suzuki.

Implement currentStackPointer for Linux/LoongArch64 to fix the build error.

* Source/WTF/wtf/StackPointer.cpp:

Canonical link: https://commits.webkit.org/276739@main
